packages/reporting/src/components/reporter-side-bar-structure.tsx

```diff
@@ -58,6 +58,11 @@ export const buildReportMenuStructure = (
       title: 'Lifts',
       onClick: () => setCurrentReport(Reports.showLiftStateReport),
     },
+    {
+      icon: <ArrowDropUpIcon />,
```

Comment:
Can you find an icon that represents the tasks better?

Reply:
Ok!